spec/controllers/v0/occupations_controller_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.